src/Components/v2/Carousel.tsx

```diff
@@ -206,6 +206,7 @@ export class BaseCarousel extends React.Component<
    */
```

Comment:
This comment should be improved to say that the lib cannot even be loaded in environments that lack a global `window` object.